pvlib/ivtools.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 29 10:34:10 2019
+
+@author: cwhanse
+"""
+
+import numpy as np
+
+
+def fit_cec_with_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                     gamma_pmp, cells_in_series, temp_ref=25):
+    '''
+    Estimates parameters for the CEC single diode model using the SAM SDK.
+
+    Parameters
+    ----------
+    celltype : str
+        Value is one of 'monoSi', 'multiSi', 'polySi', 'cis', 'cigs', 'cdte',
+        'amorphous'
+    v_mp : float
+        Voltage at maximum power point at standard test condition (STC)
+    i_mp : float
+        Current at maximum power point at STC
+    Voc : float
+        Open circuit voltage at STC
+    i_sc : float
+        Short circuit current at STC
+    alpha_sc : float
+        Temperature coefficient of short circuit current at STC, A/C
+    beta_voc : float
+        Temperature coefficient of open circuit voltage at STC, V/C
+    gamma_pmp : float
+        Temperature coefficient of power at maximum point point at STC, %/C
+    cells_in_series : int
+        Number of cells in series
+    temp_ref : float, default 25
+        Reference temperature condition
+
+    Returns
+    -------
+    a_ref : float
+        The product of the usual diode ideality factor (n, unitless),
+        number of cells in series (Ns), and cell thermal voltage at reference
+        conditions, in units of V.
+
+    I_L_ref : float
+        The light-generated current (or photocurrent) at reference conditions,
+        in amperes.
+
+    I_o_ref : float
+        The dark or diode reverse saturation current at reference conditions,
+        in amperes.
+
+    R_sh_ref : float
+        The shunt resistance at reference conditions, in ohms.
+
+    R_s : float
+        The series resistance at reference conditions, in ohms.
+
+    Adjust : float
+        The adjustment to the temperature coefficient for short circuit
+        current, in percent
```

Comment:
My understanding is that this adjustment is simultaneously applied to `beta_voc`.

Reply:
It is, but `beta_voc` is a model output, it is not used when computing an IV curve.